view/SSTemplateParser.php

```diff
@@ -3775,6 +3780,8 @@ function ClosedBlock_Handle_With(&$res) {
 		}
 		
 		$on = str_replace('$$FINAL', 'obj', ($arg['ArgumentMode'] == 'default') ? $arg['lookup_php'] : $arg['php']);
+		// obj() has a fourth argument to enable caching, so add that to the end of the call
+		$on = preg_replace('/(\))$/', ', true)', $on);
```

Comment:
Yeah, not ideal :(. I’m sure it’s possible to achieve the same output some other way, I’m just not anywhere near familiar enough with the parser to do that sort of refactoring!